src/utils.rs

```diff
@@ -264,7 +264,7 @@ mod tests {
 
     #[test]
     fn chain_checker() {
-        assert!(ChainConfig::from_str("gemini-3c").is_ok());
+        assert!(ChainConfig::from_str("gemini3c").is_ok());
```

Comment:
This is not backwards compatible. I think we can switch to `strum` once we will remove gemini-3c and switch to 3d

Reply:
good catch, but we won't be making a release for gemini-3c nevertheless. So, I think it's safe?